lib/std/zig/render.zig

```diff
@@ -200,7 +201,6 @@ fn renderMember(
                 ais.pushIndent();
                 const lbrace = tree.nodes.items(.main_token)[body_node];
                 try renderToken(r, lbrace, .newline);
-                try discardAllParams(r, fn_proto);
```

Comment:
This incorrectly discarded parameters when they were used in e.g. the return type expression. It now instead relies on the unused function parameter fixup.